File: pgm/ABC_Visual_Basic/ABC_165_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim K = Console.ReadLine()
        Dim A, B As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1)
        Console.WriteLine(If(A <= Math.Floor(CType(B, Decimal) / K) * K, "OK", "NG"))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_165_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        int K = int.Parse(Console.ReadLine());
        String[] arr = Console.ReadLine().Split();
        int A, B;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        Console.WriteLine(A <= B / K * K ? "OK" : "NG");
        return;
    }
}